tic-tac-toe.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+# Runs the Python app on Linux or macOS.
```

Comment:
1. The script is agnostic to the OS. It is written for a specific shell (Bash), not a specific OS.
2. Consider using the wording _"An application launcher for Bash"_.
3. I propose to stop emphasizing that this is a Python app in places where this fact is irrelevant.

Reply:
Done.